cvat/apps/dataset_manager/formats/yolo.py

```diff
@@ -26,12 +33,20 @@ def _export(dst_file, temp_dir, instance_data, save_images=False):
 
     make_zip_archive(temp_dir, dst_file)
 
-@importer(name='YOLO', ext='ZIP', version='1.1')
-def _import(src_file, temp_dir, instance_data, load_data_callback=None, **kwargs):
+
+def _import_common(
+    src_file,
+    temp_dir,
+    instance_data,
+    load_data_callback=None,
+    format_name="yolo",
+    import_kwargs=None,
+    **kwargs
+):
     Archive(src_file.name).extractall(temp_dir)
 
     image_info = {}
-    frames = [YoloExtractor.name_from_path(osp.relpath(p, temp_dir))
+    frames = [dm_env.extractors.get(format_name).name_from_path(osp.relpath(p, temp_dir))
```

Comment:
You should save the extractor to a variable instead of obtaining it anew for every file.

Reply:
done